datasetinsights/io/downloader/unity_simulation.py

```diff
@@ -0,0 +1,116 @@
+import logging
+import os
+import re
+from pathlib import Path
+
+from datasetinsights.io.downloader.base import DatasetDownloader
+from datasetinsights.io.usim import Downloader, download_manifest
+
+logger = logging.getLogger(__name__)
+
+
+class UnitySimulationDownloader(DatasetDownloader):
+    """ Downloads unity simulation datasets
+        Args:
+        access_token: "Access token to be used to authenticate to
+         unity simulation for downloading the dataset"
+
+    """
+
+    PROTOCOL = "usim://"
+
+    def __init__(self, access_token=None):
+        self.access_token = access_token
+        self.run_execution_id = None
+        self.project_id = None
+
+    def download(self, source_uri, output, include_binary):
+        """
+
+        Args:
+            source_uri: This is the downloader-uri that indicates where on
+             unity simulation the dataset should be downloaded from.
+
+            output: This is the path to the directory
```

Comment:
Have you build documentation to check if this docstring is parsed properly?

Reply:
No not yet. Before all conversations are resolved